include/yaclib/fault/random.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <yaclib/fault/detail/random/random_device.hpp>
+
+#include <random>
+#include <string>
+
+namespace yaclib_std::random {
+
+#ifdef YACLIB_FAULT_TEST_ON
+
+using random_device = yaclib::detail::RandomDevice;
```

Comment:
Why do we override std random in thread fault injection case?

Reply:
Because I tried to make some parts common for thread and fiber fault injection if possible